### DISCUSSION TITLE
Is one-way communication allowed in Client-Server?

### DISCUSSION THREAD

**Discussion by wzhao18** [eclipse-iceoryx/iceoryx#2190] [General]
Hi there, I have a question. Some help would be greatly appreciated!

I am currently using the architecture shown in https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/request_response/client_cxx_untyped.cpp In my use case, some client requests expect a response while certain do not. My program does not function correctly if I comment out the response part. I would like to know how to do this properly.

Thank you in advance!

**Comment by elBoberido** [comment]
@wzhao18 what exactly happens when you comment out that code? It's a while since I touched that code but AFAIR it should work.

**Reply by wzhao18** [reply]
Thanks for getting back and confirming that it is supposed to work! I will do some fine-grained testing and hopefully figure out where the bug comes from.
